bucket/gdu.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "4.10.1",
+    "description": "Disk usage analyzer with console interface written in Go",
```

Language is completely irrelevant for end user
```suggestion
    "description": "Disk usage analyzer with console interface",
```